dropwizard-jersey/src/test/java/io/dropwizard/jersey/AbstractJerseyTest.java

```diff
@@ -12,7 +12,7 @@
 public abstract class AbstractJerseyTest extends JerseyTest {
 
     private static final int DEFAULT_CONNECT_TIMEOUT_MS = 1000;
-    private static final int DEFAULT_READ_TIMEOUT_MS = 5000;
+    private static final int DEFAULT_READ_TIMEOUT_MS = 500000;
```

Comment:
Good catch, thanks!

This change was required for debugging Jersey-based without running into the 5 seconds timeout every time. It will be reverted before merging this PR. 😉